docs/sidebars.ts

```diff
@@ -386,18 +386,10 @@ module.exports = {
           "type": "doc",
           "id": "manuals/developer/runtimes"
         },
-        {
-          "type": "doc",
-          "id": "manuals/developer/package-managers"
-        },
```

Comment:
Do we need redirects?

Reply:
I considered it but did not implement because
- this page was added just a couple of days ago so less likely to have been indexed
- once the merged manual is live all the current URLs and existing redirects will become invalid anyway